.circleci/config.yml

```diff
@@ -34,18 +34,19 @@ commands:
           echo 'Changes detected:'
           echo $changes
 
-          if [[ $changes ]]; then
-            echo 'export ROOT_CHANGE=true' >> $BASH_ENV
+          if [[ $changes ] || [ $CIRCLE_BRANCH == "master" ]]; then
```

Comment:
What does this extra part do?

Looks like it needs to be `[[ $changes ]] || [[ $CIRCLE_BRANCH == "master" ]]`